### DISCUSSION TITLE
How use referencedTable option with or in a url ?

### DISCUSSION THREAD

**Discussion by bkan36** [supabase/supabase#29148] [Questions] [answered]
here is my suspabase js function query that works
I want do the same in a supabase URL

```
 const { data, error } = await this.supabase
                .from('user_club')
                .select('user:user_id!inner(id, name, lastname, pseudo, player!inner(position))')
                .eq('club_id', clubId)
                .or(`name.ilike.%${searchQuery}%, lastname.ilike.%${searchQuery}%`, { referencedTable: 'user_id' })
                .limit(limit);

```

I tried to translate this to an sql query with chatGPT then retranslate the SQL query with this supabase tools:

https://supabase.com/docs/guides/api/sql-to-rest

but it doesn't works

Even with their own example from the doc:

https://supabase.com/docs/reference/javascript/or

.../rest/v1/countries?select=country_name:name,...cities!inner(city_name:name)&or=(id.eq.1,cities.name.eq.Beijing)

I've this kind of error
{
  "code": "PGRST100",
  "details": "unexpected \"a\" expecting \"not\" or operator (eq, gt, ...)",
  "hint": null,
  "message": "\"failed to parse logic tree ((id.eq.1,cities.name.eq.Beijing))\" (line 1, column 19)"
}

I think the problem come from the referencedTable,  I don't know how to set this option in the url,

thanks

**Comment by GaryAustin1** [comment, marked as answer]
https://docs.postgrest.org/en/v12/references/api/resource_embedding.html#embedded-filters
So you would do &user_id.or=(......)

**Reply by bkan36** [reply]
Thank you very much it works using user.or...
